pandas/tests/plotting/test_series.py

```diff
@@ -622,6 +622,7 @@ def test_kde_kwargs(self):
             pytest.skip("mpl is not supported")
 
         from numpy import linspace
+        _check_plot_works(self.ts.plot.kde, bw_method=None, ind=20)
```

Comment:
Could you add a test with `np.int(20)` as well.

Reply:
can you add a use with ``bw_method`` as a string.